lib/MusicBrainz/Server/Data/WorkType.pm

```diff
@@ -38,6 +38,14 @@ sub in_use {
         $id);
 }
 
+sub get_description
```

Comment:
You should delete this, then use `$c->model('WorkType')->get_all` in `Controller::Work` to create a hash of work type descriptions by ID. Then place that hash in `$c->stash` to access it from the template.